api/updatepb/status.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package update;
+
+option go_package = "api/updatepb;updatepb";
+
+import "protoc-gen-openapiv2/options/annotations.proto";
+import "google/api/annotations.proto";
+
+message ReadyRequest {}
+
+message ReadyResponse {}
```

Comment:
I missed this thing, when updater is not ready we returning an error?

Reply:
At the moment we never return an error. We will though if there's some additional logic in the future.
`managed` expects no error in case `pmm-server-upgrade` is ready.